gateway-service-definitions/src/main/resources/services/trinoui/365/service.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<service role="TRINOUI" name="trino" version="365">
```

Comment:
Missing the metadata information here. Without that piece of data, the Trino UI won't show up on the Knox Home page. Sample metadata section for Ranger:
https://github.com/apache/knox/blob/master/gateway-service-definitions/src/main/resources/services/rangerui/1.0.0/service.xml#L18-L23

Reply:
Good point @smolnar82!